bin/list-all

```diff
@@ -5,15 +5,23 @@ function sort_versions() {
     LC_ALL=C sort -gt. -k 1,1 -k 2,2n -k 3,3n -k 4,4n -k 5,5n | sort -r - | awk '{print $2}'
 }
 
-cmd="curl -Ls0"
-releases_path="https://api.github.com/repos/JuliaLang/julia/releases"
+releases=$(tempfile -s .json)
+headers=$(tempfile)
```

Comment:
`tempfile` is not available on non-debian-based linux distributions. Thus, this PR broke asdf-julia for Arch Linux, Alpine et al.

Reply:
Ah, pity. I guess `mktemp` would do the trick then, wouldn't it? I'll just replace it with that, then.

Reply:
I already did that [here](https://github.com/rkyleg/asdf-julia/pull/26). But if you could test whether that PR works on your machine that would be helpful. I don't have access to Ubuntu right now.

Reply:
See here https://github.com/rkyleg/asdf-julia/pull/27.

Reply:
:-) Ok then. I dropped my PR, fetched yours and tested it on my machine. Good news is, tt works just fine.